### PR TITLE
Add initial Go ESB implementation

### PR DESCRIPTION
**Pull request by nullxDEADBEEF**
* Add Go version of the ESB

### CODE REVIEW

File: go_esb/main.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	c.JSON(http.StatusForbidden, gin.H{"info": "Invalid token"})
+}
+
+func transformMessage(c *gin.Context, message Message, format string) {
+	switch format {
+	case "JSON":
+		JSONTransformer(c, message)
+	case "XML":
+		XMLTransformer(c, message)
+	case "YAML":
```
Comment (AndreasPB) [inline go_esb/main.go]: Make it accept YML pls :D

File: go_esb/main.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	},
+}
+
+var messages = map[string][]Message{
+	"1": {
+		{
+			Id: "1a76658a-7e4c-4f24-9a96-f68ef3526008", Content: "I am message 1",
+		},
+		{
+
```
Comment (AndreasPB) [inline go_esb/main.go]: Very OCD triggering newline

**Review by AndreasPB — APPROVED** [review]

File: go_esb/main.go (hunk 1 above)
Comment (nullxDEADBEEF) [reply go_esb/main.go]: Done

Comment (AndreasPB) [reply go_esb/main.go]: Oh, meant both :P

File: go_esb/main.go (hunk 2 above)
Comment (nullxDEADBEEF) [reply go_esb/main.go]: Made the messages more compact now

File: go_esb/main.go (hunk 1 above)
Comment (nullxDEADBEEF) [reply go_esb/main.go]: Ahhh :D

Comment (nullxDEADBEEF) [reply go_esb/main.go]: Accepts both now :D

**Pull request merged**